10_install_server/12_install_root_server_rhel.txt

```diff
@@ -0,0 +1,99 @@
+=== Install Rudder Root server on Enterprise Linux ( RedHat or RedHat-like )
+
+==== Add the Rudder packages repository
+
+Run the following command as root:
+
+Note: <version> is your RHEL version
+
+----
+
+echo "[Rudder_3.0]
+name=Rudder 3.0 EL repository
+baseurl=http://www.rudder-project.org/rpm-3.0/RHEL_<version>/
+gpgcheck=1
+gpgkey=http://www.rudder-project.org/rpm-3.0/RHEL_<version>/repodata/repomd.xml.key" > /etc/yum.repos.d/rudder.repo
+
+----
+
+==== Install your Rudder Root Server
+
+To begin the installation, you should simply install the rudder-server-root
+metapackage, which will install the required components:
+
+----
+
+yum install rudder-server-root
+
+----
+
+Now jump to <<initial-config,the next section to configure your server>>.
+
+[WARNING]
+
+====
+
+Rudder server doesn't support SELinux yet (see http://www.rudder-project.org/redmine/issues/2882),
+so you should set it to be permissive with these commands:
+
+----
+
+sed -i "s%^\s*SELINUX=.*%SELINUX=disabled%" /etc/sysconfig/selinux
+setenforce 0
+
+----
+====
+
+[WARNING]
+
+====
+
+The webapp runs on HTTPS/443 port so you need to allow access to your server on this port.
+The file to edit is /etc/sysconfig/iptables on EL6 and should look like 
```

Comment:
Please add the keyword "firewall" in here somewhere - users sometimes search for this keywork but fail to find it.